app/Http/Controllers/TicketController.php

```diff
@@ -36,13 +36,19 @@ public function show($id)
         //We need to ensure that this ticket belongs to this user
         $tickets = $this->getTickets();
         foreach ($tickets as $ticket) {
-            if ($ticket->getTicketID() == $id) {
-                $accountTicketController = new AccountTicketController();
-                $replies = array_reverse($accountTicketController->getReplies($ticket, 1));
-                //Clear the cache here, because you may see a ticket with ISP responses but the list may not show it yet
+            try {
+                if ($ticket->getTicketID() == $id) {
+                    $accountTicketController = new AccountTicketController();
+                    $replies = array_reverse($accountTicketController->getReplies($ticket, 1));
+                    //Clear the cache here, because you may see a ticket with ISP responses but the list may not show it yet
+                    $this->clearTicketCache();
+                    return view("pages.tickets.show", compact('replies', 'ticket'));
+                }
+            }catch (Exception $e) {
+                Log::error($e->getMessage());
```

Comment:
You can actually log the entire exception with `report($e)`. Just logging the error does not yield very good debugging information, but maybe it's okay... @SonarSoftware ?

Reply:
The `$e->getMessage()` was following the error handling standard already built in the app. I do agree with debugging information and don't mind changing it.

Reply:
Yeah, it'll probably need to be addressed throughout the rest of the application as well. Could probably set this up as a separate PR.